fuse_core/test/test_timestamp_manager.cpp

```diff
@@ -542,7 +554,7 @@ TEST_F(TimestampManagerTestFixture, MultiSegment)
   ASSERT_EQ(0ul, generated_time_spans.size());
 }
 
-TEST_F(TimestampManagerTestFixture, MultiSegmentBeforeBeginning)
+TEST_F(TimestampManagerTestFixture, MultiSegmentBeforBeginning)
```

Comment:
Did I lose an 'e' on purpose?
```suggestion
TEST_F(TimestampManagerTestFixture, MultiSegmentBeforeBeginning)
```

Reply:
Fixed :smile: